mindsdb/integrations/handlers/kraken_handler/kraken_tables.py

```diff
@@ -0,0 +1,50 @@
+import time
+
+from mindsdb.integrations.libs.api_handler import APITable
+
+
+class KrakenTradesTable(APITable):
+    def select_trade_history(self, queries: dict):
```

Comment:
This method should be called `select()`. In order to implement the SQL `SELECT` functionality for any `APITable` implementation, this method is required. Similar methods should also be implemented for the other SQL operations (`insert()`, `update()` and `delete()`). Please check the `APITable` class.

Reply:
Once you have fixed this, please post a screenshot or a video of running a `SELECT` statement against this table through the MindsDB SQL Editor.